### DISCUSSION TITLE
Current trigger Q1 Mosfet options ?

### DISCUSSION THREAD

**Discussion by ghost** [darthcloud/BlueRetro#725] [Q&A] [answered]
The current trigger use the YJS2301A transistor.
Is there an alternative for YJS2301A ? 
This transistor is only avalible on Digikey on a minimun quantity of 3000 ($201). not mentioning the shipping fee, $78 for Brazil.

**Comment by konwektor** [comment, marked as answer]
Shame on You. Here  orders from  5pcs, about 6$ shipping, and maybe 4$ duty customs to Brazil [https://www.lcsc.com/product-detail/MOSFETs_Yangzhou-Yangjie-Elec-Tech-YJS2301A_C699284.html](url)

**Reply by ghost** [reply]
Thank you so much !!! I'm gonna buy it right now
